tests/safety/test_subaru.py

```diff
@@ -73,5 +73,19 @@ def _pcm_status_msg(self, enable):
     return self.packer.make_can_msg_panda("CruiseControl", 0, values)
 
 
+class TestSubaruForesterHybridSafety(TestSubaruSafety):
+
+  def setUp(self):
+    self.packer = CANPackerPanda("subaru_global_2017_generated")
+    self.safety = libpandasafety_py.libpandasafety
+    self.safety.set_safety_hooks(Panda.SAFETY_SUBARU, Panda.FLAG_SUBARU_FORESTER_HYBRID)
+    self.safety.init_tests()
+
+  def _pcm_status_msg(self, enable):
```

Comment:
already checked for a common cruise activated signal?

Reply:
No, need to get a route first from somebody.